Branch for added features

ClintonYounge opened the pull request:
Added-

1. All code necessary to implement a responsive home and about page.
2. A mobile menu that'll allow the visitor to switch between pages.
3. A section of the home page made dynamically with JavaScript.

ClintonYounge committed:
Create .gitignore files

Add node_modules to gitignore

ClintonYounge committed:
Add navbar

ClintonYounge committed:
Day 1- Update html css and js

Add-
-responsive home page.
-guides section dynamically created with js.
-mobile menu.
-Add about page with mobile menu.

ClintonYounge committed:
Day 2- Update html css and js

-fixed menu.
-completed basic about page.
-completed to be optimized mobile versions.
-fix all links for mobile version

ClintonYounge committed:
Day 3- Update html css and js

Changes-
-Added the social section above the desktop nav.
-added the footer in the desktop versions.
-Used media query to change the content of html in desktop version.
-add logo to nav in desktop version.

ClintonYounge committed:
Update readme file

ClintonYounge committed:
Fix styelint and eslint errors

ClintonYounge committed:
Fix stylelint errors

ClintonYounge committed:
Fix path errors, Improve accessibility

zieeco commented on README.md:
```diff
@@ -6,79 +8,140 @@
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Below is a link to the Live Demo of my project.
+- [Experience Guyana](coming soon)
```
- Let's kindly deploy the this website using Github pages to meet this part of the project requirement.

zieeco commented on README.md:
```diff
@@ -6,79 +8,140 @@
+> Below is a link to the Live Demo of my project.
+- [Experience Guyana](coming soon)
+
+> Below is a link to Loom explaining my project.
+- [My Loom Video](coming soon.)
 
```
- Please, let's kindly add the presentation video. Please, check the project requirement to follow the instructions regarding adding the video to meet this part of the project requirement.

zieeco commented on README.md:
```diff
@@ -6,79 +8,140 @@
 
-I would like to thank my standup team for help settling my mind to persevere through this project.
+> Many thanks to [Microverse](https://www.microverse.org/) for teaching me all I know about programming.
 
```
- Let's also give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) to meet this part of the project requirement.

zieeco commented on index.html:
```diff
@@ -0,0 +1,191 @@
+                </section>
+            </div>
+        </section>
+    </section>
```
### THIS CHANGE IS APPLICABLE TO THE ABOUT PAGE ALSO

- Hey, Please, let's kindly improve our page structure by doing the following to enforce a better way of having a semantic index page below:

- [ ] Kindly remove the <header> from this section as the header hold the introductory contents of your website so it should be a stand-alone element not wrapped in a section. also, the header should be the first element in the body before another element of your page.
- [ ] The   `<div class="social">` should be moved into the nav inside the header as this is a nav item.

zieeco commented on index.html:
```diff
@@ -0,0 +1,191 @@
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </div>
```
### THIS CHANGE IS APPLICABLE TO THE ABOUT PAGE ALSO


- Please, let's follow the design properly paying attention to the background color, and the colors of the icons.

*design*
![Screenshot from 2023-02-16 22-04-52](https://user-images.githubusercontent.com/59397159/219486695-3a78ab78-9bc7-4215-b1e8-0f5991ab0054.png)

*your design*
![Screenshot from 2023-02-16 22-04-26](https://user-images.githubusercontent.com/59397159/219486702-9f47eaa9-f20b-422b-acfa-05e2a8d1003e.png)

zieeco commented on index.html:
```diff
@@ -0,0 +1,191 @@
+                <div class="menu-icon">
+                    <box-icon name='menu' size="lg" class=".hamburger"></box-icon>
+                </div>
+                <li class="logo"><img src="./images/left-logo.png" alt=""></li>
```
- Please, kindly use another element like a div to wrap the logo as the li is ideal to be used with the ul as it signifies a list item. using the li alone is not semantically correct.

zieeco commented on about-page/about.html:
```diff
@@ -0,0 +1,134 @@
+                <div class="about-menu-icon">
+                    <box-icon name='menu' size="lg" class=".hamburger"></box-icon>
+                </div>
+                <li class="logo"><img src="images/left-logo.png" alt=""></li>
```
- Please, kindly use another element like a div to wrap the logo as the li is ideal to be used with the ul as it signifies a list item. using the li alone is not semantically correct.
- Also, the logo is not rendering, kindly fix it.

zieeco commented on about-page/about.html:
```diff
@@ -0,0 +1,134 @@
+            <div class="image2">
+                <a href="#"><img src="/images/Experience-Guyana-Logo3.png" alt="Experience Guyana previous year logo"></a>
+            </div>
+        </div>
```
- Hey, Please, let's improve our design on each card to match the design as much as we can.
   - The date should come first before the next content
   - Improve the color of each card
   - The date and content have a white color.
   - see the image below to compare your design


*design*
![Screenshot from 2023-02-16 22-19-06](https://user-images.githubusercontent.com/59397159/219489757-d58a24f9-9a2f-4727-a139-d0f0ad6334b1.png)

*your design*
![Screenshot from 2023-02-16 22-18-43](https://user-images.githubusercontent.com/59397159/219489768-3d4847ff-f88b-4ed3-858c-3275a64c27df.png)

zieeco commented on about-page/about.html:
```diff
@@ -0,0 +1,134 @@
+                <p>2023, some rights reserved</p>
+            </div>
+        </div>
+    </footer>
```
## THIS CHANGE IS APPLICABLE TO THE HOME PAGE `index.html` 

- Let's also improve on the footer, the highlighted part of the image below should be the logo of your webpage used in the home page [index.html]. If you must use the text content in the placement of your logo, then kindly change the logo on the homepage [index.html] to be the same to improve your design.

![Screenshot from 2023-02-16 22-26-15](https://user-images.githubusercontent.com/59397159/219491200-4c5f446a-20ff-494d-b4f3-e1a00e30dc5d.png)

zieeco commented on index.html:
```diff
@@ -0,0 +1,191 @@
+    </section>
+
+    <footer>
+        <div class="footer-top">
+            <h2>Partners</h2>
+            <div class="partners">
+                <ul>
+                    <li><img src="/images/Exxon-Mobil.jpg" alt=""></li>
+                    <li><img src="/images/Massy-logo.jpg" alt=""></li>
+                    <li><img src="/images/Banks-DIH.jpg" alt=""></li>
+                    <li><img src="/images/Ministry-Of-Tourism.jpg" alt=""></li>
+                </ul>
+            </div>
+        </div>
```
- Hey, Please, kindly note that this part of the webpage is a standalone section. it's a section of the partners of the page and it is not part of the footer. please, move it out from the footer and make it a section and not a div
   - Let's use the correct background color
   - The correct color of  the text on the logo
   - Add the missing red horizontal line under the word `Partner`
   - Let's use the correct spacing between the first and last logo
   - see the image below to compare your design

*your design*
![Screenshot from 2023-02-16 22-40-00](https://user-images.githubusercontent.com/59397159/219493196-8452dccf-796b-4818-b9d9-47a8adb4d6eb.png)
*original design*
![Screenshot from 2023-02-16 22-39-30](https://user-images.githubusercontent.com/59397159/219493208-c2cdc3d0-5480-409f-9f72-39af10e8d109.png)

zieeco replied:
### MISSING FOOTER ON THE MOBILE VERSION OF THE HOMEPAGE


- Please, kindly make the footer visible in the mobile version of the homepage

zieeco commented on css/style.css:
```diff
@@ -0,0 +1,293 @@
+  flex-direction: column;
+  padding: 20px 0;
+  padding-bottom: 50px;
+}
```
- Let's kindly improve our style by making some adjustments to the spacing at the left and right of this section. see the images below:


*original design*
![Screenshot from 2023-02-16 22-44-41](https://user-images.githubusercontent.com/59397159/219494661-9fd63c98-c647-4979-af67-7f9e4f50c847.png)

*your design*
![Screenshot from 2023-02-16 22-43-59](https://user-images.githubusercontent.com/59397159/219494674-659b6c97-c41c-411d-814a-6903e4241e95.png)

zieeco commented on css/style.css:
```diff
@@ -0,0 +1,293 @@
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: auto;
+}
```
- Hey, please, let's improve our style to match the design as much as we can below:
   - Use the correct background color on the cards and the main background 
   - Align the program correctly on the desktop version
   - Add the missing white border hover effect on each card
   - see the images below to compare your design


*original design*
![Screenshot from 2023-02-16 22-51-23](https://user-images.githubusercontent.com/59397159/219495819-93aac5f3-f3d1-4f12-8bf1-77877d9e2435.png)


*your design*
![Screenshot from 2023-02-16 22-58-38](https://user-images.githubusercontent.com/59397159/219496369-1fd7e088-b740-41dc-85fb-ac3bc60fa31b.png)

zieeco reviewed (changes requested):
Hi @ClintonYounge ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._


<img src="https://media0.giphy.com/media/8UGVQESpDMMQl7yd3F/giphy.gif"/>


## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ClintonYounge committed:
Update html css and js

-Edit to match design
-Update readme to include design author.
-Changed background images/
-Update main program layout.
-Increase responsiveness.

ClintonYounge committed:
Update js and css

-Update images
-edit image sizes
-edit image paths
-fix eslint and stylint linter errors

ClintonYounge committed:
Update images path and size

ClintonYounge committed:
Fix linter error

ClintonYounge committed:
Update images path

ClintonYounge committed:
Update nav items and nav css

ClintonYounge committed:
update popup menu  margin

Applied to the about page

ClintonYounge committed:
Update html and css

Increase overall responsiveness by utilizing media queries for 4 separate breakpoints

ClintonYounge committed:
Add title underline, update readme

ClintonYounge committed:
Fix stylelint spacing errors

ClintonYounge committed:
Fix stylelint spacing error

arslanbisharat commented:
Hi @ClintonYounge,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
